result/uds.yaml

```diff
@@ -0,0 +1,101 @@
+components:
+  schemas:
+    User.Defined.Metrics.Request:
+      description: |-
+        The user defined metrics result request to the traffic generator
+      type: object
+      properties:
+        choice:
+          description: |-
+            The type
+          type: string
+          x-field-uid: 1
+          x-enum:
+            port_names:
```

Comment:
why not use PortMetrics.Association here?

Reply:
we removed column names and now we are reusing association